comps/guardrails/src/guardrails/integrations/llamaguard.py

```diff
@@ -46,16 +45,23 @@ def get_unsafe_dict(model_id=DEFAULT_MODEL):
         }
 
 
-def get_tgi_service_model_id(endpoint_url, default=DEFAULT_MODEL):
-    """Returns Hugging Face repo id for deployed service's info endpoint
-    otherwise return default model."""
+def get_service_model_id(endpoint_url, default=DEFAULT_MODEL):
+    """Returns model_id from the OpenAI-compatible /v1/models endpoint.
+
+    Falls back to default if the request fails or no models are returned.
+    """
     try:
         requests = JsonRequestsWrapper()
-        info_endpoint = os.path.join(endpoint_url, "info")
-        model_info = requests.get(info_endpoint)
-        return model_info["model_id"]
+        proxies = {"http": None, "https": None}
+        models_endpoint = os.path.join(endpoint_url.rstrip("/"), "v1/models")
+        print(models_endpoint)
```

Comment:
Should we use logger instead of print here?

Reply:
test code, just removed